current-release-tags.sh

```
@@ -0,0 +1,31 @@
+    | jq -r '.content_stream_tags[]'
+}
+
+# Method 2: Use data from the Cicada repo
+function content-stream-tags-from-cicada() {
+  curl -s \
+    "https://gitlab.cee.redhat.com/releng/pyxis-repo-configs/-/raw/main/products/rhtas/rhtas.yaml" \
+    | yq -r --yaml-fix-merge-anchor-to-spec '.repositories[].repository|select(.repository=="rhtas/ec-rhel9").content_stream_tags[]'
+}
+
+# Two ways to do it:
+case "{1-:""}" in
```

Shellcheck (0.11.0)

[warning] This word is constant. Did you forget the $ on a variable?

(SC2194)